hs/src/Reach/Connector/ALGO.hs

```diff
@@ -2776,7 +2776,7 @@ ct = \case
 
 -- Reach Constants
 reachAlgoBackendVersion :: Int
-reachAlgoBackendVersion = 10
+reachAlgoBackendVersion = 11
```

Comment:
This is not needed. You only do this when you change the connector representation